lib/manageiq_performance/middlewares/active_record_queries.rb

```diff
@@ -130,12 +132,19 @@ def should_measure?
           Thread.current[:miq_perf_sql_query_data]
         end
 
+        def include_queries?
+          return @include_queries if defined?(@include_queries)
+          @include_queries = ManageIQPerformance.config.include_sql_queries?
```

Comment:
Huh, I guess I thought that `||=` was a `.nil?` check and not a "truthy" check.  Works for me.